README.md

```diff
@@ -242,10 +238,84 @@ stylesheet](inst/extdata/xml2md_gfm.xsl) and provide its filepath as
 
 ### The special case of tables
 
-  - Tables are supposed to remain/become pretty after a full loop
+-   Tables are supposed to remain/become pretty after a full loop
     `to_xml` + `to_md`. If you notice something amiss, e.g. too much
     space compared to what you were expecting, please open an issue.
 
+### LaTeX equations
+
+While markdown parsers like pandoc know what LaTeX is, commonmark does
+not, and that means LaTeX equations will end up with extra markup due to
+commonmark’s desire to escape characters.
+
+For users of the `yarn` object, if you have LaTeX equations that use
+either `$` or `$$` to delimit them, you can protect them from formatting
+changes with the `$protect_math()` method:
+
+``` r
+path <- system.file("extdata", "math-example.md", package = "tinkr")
+math <- tinkr::yarn$new(path)
+math$tail() # malformed
+#| 
+#| $$
+#| Q\_{N(norm)}=\\frac{C\_N +C\_{N-1}}2\\times
+#| \\frac{\\sum *{i=N-n}^{N}Q\_i} {\\sum*{j=N-n}^{N}{(\\frac{C\_j+C\_{j-1}}2)}}
+#| $$
+math$protect_math()$tail() # success!
+#| 
+#| $$
+#| Q_{N(norm)}=\frac{C_N +C_{N-1}}2\times
+#| \frac{\sum _{i=N-n}^{N}Q_i} {\sum_{j=N-n}^{N}{(\frac{C_j+C_{j-1}}2)}}
+#| $$
+```
+
+Note, however, that there are a few caveats for this:
+
+1.  The dollar notation for inline math must be adjacent to the text.
+    E.G. `$\alpha$` is valid, but `$ \alpha$` and `$\alpha $` are not
+    valid.
+
+2.  We do not currently have support for bracket notation
+
+3.  If you use a postfix dollar sign in your prose (e.g. BASIC commands
+    or a Burroughs-Wheeler Transformation demonstration), you must be
+    sure to either use punctuation after the trailing dollar sign OR
+    format the text as code. (i.e. `` `INKEY$` `` is good, but `INKEY$`
+    by itself is not good and will be interepreted as LaTeX code,
+    throwing an error:
+
+    ``` r
+    path <- system.file("extdata", "basic-math.md", package = "tinkr")
+    math <- tinkr::yarn$new(path)
+    math$head(15) # malformed
+    #| ---
+    #| title: basic math
+    #| ---
+    #| 
+    #| BASIC programming can make things weird:
+    #| 
+    #| - Give you $2 to tell me what INKEY$ means.
+    #| - Give you $2 to *show* me what INKEY$ means.
+    #| - Give you $2 to *show* me what `INKEY$` means.
+    #| 
+    #| Postfix dollars mixed with prefixed dollars can make things weird:
+    #| 
+    #| - We write $2 but say 2$ verbally.
+    #| - We write $2 but *say* 2$ verbally.
+    math$protect_math() #error
+    #| Error: Inline math delimiters are not balanced.
+    #| 
+    #| HINT: If you are writing BASIC code, make sure you wrap variable
+    #|       names and code in backtics like so: `INKEY$`. 
+    #| 
+    #| Below are the pairs that were found:
+    #|            start...end
+    #|            -----...---
+    #|  Give you $2 to ... me what INKEY$ means.
+    #|  Give you $2 to ... 2$ verbally.
+    #| We write $2 but ...
+    ```
+
```

Comment:
it does! I just wonder what folks who do not use the yarn object should do: probably update their code to use the yarn object?

Reply:
I think the reason why I am using the yarn object here is because we can use our namespace object in the internal parsing, but I can export that object so that we can use it as default.

```
structure(c(md = "http://commonmark.org/xml/1.0"), class = "xml_namespace")
```